packages/fuels-test-helpers/src/chains.rs

```diff
@@ -53,35 +50,50 @@ async fn no_op_signed_transaction(wallet: &LocalWallet) -> Result<Transaction, E
     Ok(transaction)
```

Comment:
The thing is I would love to do that, but Transaction is an Enum of two variants and you can't index into the fields of the struct-like variants like this. You need to know if it is a Script or a Transaction. 

And to know that you have to match. And if you match you need to handle the other case.

This would work if the variants weren't struct-like but rather contained structs in them like:
```
enum Transaction {
Script(ScriptData), 
...
}
```
Then you could match and extract the script data and do this on it since you know the exact type.

But the thing is this code is in `fuel-tx` and I didn't fancy doing breaking changes for a few lines here.

If you can find a better way to do this I would absolutely love this, cuz this is hideous.

Please reopen if you think of something.

Reply:
Didn't try anything, will maybe toy around with it in the future haha, sorry I thought this would work lol.